letsdrill.rb

```diff
@@ -1,24 +1,43 @@
 def get_letter_grade(integer)
- 
+
   #Put your code here!
 
+	case integer
+		when 90..100 then return "A"
+		when 80..89 then return "B"
+		when 70..79 then return "C"
+		when 60..69 then return "D"
+		when 0..59 then return "F"
+	end
+
 end
 
 def shortest_string(array)
 
   #Put your code here!
 
-end
- 
+	if (array.empty?) then return nil
+	end
+
+	shortest_string = array[0]
 
+	array.each do |string|
+		if (string.length < shortest_string.length)
+			then shortest_string = string
+		end
+	end
+
+	return shortest_string
+
+end
```

Comment:
This works, but you're re-implementing a lot of work that's already done for you. Check out Enumerable#sort_by and Enumerable#min_by. Also: this is nitpicky, but Ruby style is generally to indent with 2 spaces.